FluentTerminal.App.Services/Implementation/DefaultValueProvider.cs

```diff
@@ -259,6 +259,14 @@ public ICollection<KeyBinding> GetDefaultKeyBindings(Command command)
                         Alt = true,
                         Shift = false,
                         Key = (int)ExtendedVirtualKey.Enter
+                    },
+                      new KeyBinding
+                    {
+                        Command = nameof(Command.ToggleFullScreen),
+                        Ctrl = false,
+                        Alt = false,
+                        Shift = false,
```

Comment:
There's no need to put in the `false` values. This is the default if not specified and it's easier to read the definition without them.